pkgs/development/libraries/ogre/default.nix

```diff
@@ -8,11 +8,11 @@
 , nvidia_cg_toolkit }:
 
 stdenv.mkDerivation {
-  name = "ogre-1.9.0";
+  name = "ogre-1.9-hg-20160322";
```

Comment:
It's not on the main hg branch, but on the maintenance branch for 1.9.x. Except as far as I can tell, they've never actually put together any 1.9 releases other than 1.9.0. Is ogre-1.9-hg-2016-03-22 acceptable? Maybe ogre-1.9+hg-2016-03-22?